blocklist-call/.env

```diff
@@ -1,4 +1,4 @@
-# description: The list of numbers you want to reject
+# description: Calls from these numbers will be blocked
```

Comment:
I'm wondering if we should use the longer description you provided below for the comma separated, e164 description.

Might be worth adding the link to https://www.twilio.com/docs/glossary/what-e164 for anytime we need a phone number to be entered by the user